core/src/main/scala/codesearch/core/search/Search.scala

```diff
@@ -214,15 +210,29 @@ object Search {
     * @param result matched code snippet
     */
   private[search] final case class CSearchResult(
-      pack: Package,
-      result: CodeSnippet
-  )
+                                                  pack: Package,
+                                                  result: CodeSnippet
+                                                )
 }
 sealed trait Response
 final case class SearchByIndexResult(lists: List[String], error: ErrorResponse)
```

Comment:
The `SearchByIndexResult` not used anywhere.

Reply:
But i'm using it in my code in `def search`. 
`  private def csearch(request: SearchRequest): SearchByIndexResult = {
    val indexDir = cindexDir.indexDirAs[String]
    val env      = ("CSEARCHINDEX", indexDir)
    var stderr   = new String
    val log      = ProcessLogger((o: String) => o, (e: String) => stderr = e)
    val test = for {
      _       <- logger.debug(s"running CSEARCHINDEX=$indexDir ${arguments(request).mkString(" ")}")
      results <- IO((Process(arguments(request), None, env) #| Seq("head", "-1001")).lineStream_!(log).toList)
    } yield SearchByIndexResult(results, ErrorResponse(stderr))
    test.unsafeRunSync()
  }`

Reply:
@Kabowyad Okey. Sorry my mistake =(